composer.json

```
@@ -55,6 +55,7 @@
         "phpunit/phpunit": "^9.5.13",
         "psalm/plugin-phpunit": "^0.16",
         "psalm/plugin-symfony": "^3.0",
+        "rector/rector": "dev-main",
```

I would like to use ^0.12 again, but this needs to be fixed first: https://github.com/rectorphp/rector/issues/7141

---

```suggestion
        "rector/rector": "^0.12",
```

To apply when the above is fixed.